### PR TITLE
[hi] Localize glossary entry for “API-initiated eviction”

### PR DESCRIPTION
**Pull request by shelar1423**
fix issue: #35938

### CODE REVIEW

**Comment by dipesh-rawat** [comment]
/retitle [hi] Localize glossary entry for “API-initiated eviction”

**Comment by shelar1423** [comment]
@bishal7679  @rolfedh  have a review.

**Comment by dipesh-rawat** [comment]
@shelar1423  Thank you so much for your contribution! I appreciate your effort in helping with the localization of the website. However, I would like to point out that the current way the changes have been implemented is not correct. This PR adds the "API-initiated eviction" glossary entry file content in English instead of localizing(translating) it in Hindi, which I believe is the original intention as per the issue request.

**Comment by dipesh-rawat** [comment]
/hold

Currently, PR is incorporating the English file content as-is into the Hindi localization, which is not in line with desired outcome of the issue https://github.com/kubernetes/website/issues/35938

(OK to unhold once that's corrected)

**Comment by shelar1423** [comment]
> @shelar1423 Thank you so much for your contribution! I appreciate your effort in helping with the localization of the website. However, I would like to point out that the current way the changes have been implemented is not correct. This PR adds the "API-initiated eviction" glossary entry file content in English instead of localizing(translating) it in Hindi, which I believe is the original intention as per the issue request.

understood will change it

**Comment by shelar1423** [comment]
@dipesh-rawat  so i need to change the content language to hindi and then update this?

**Comment by dipesh-rawat** [comment]
> so i need to change the content language to hindi and then update this?

@shelar1423 Certainly, go ahead with the localization/translation of the file content into Hindi, and then push those updates here.

**Comment by shelar1423** [comment]
@dipesh-rawat done with the changes

**Comment by shelar1423** [comment]
is this fine @dipesh-rawat ?

File: content/hi/docs/reference/glossary/api-eviction.md
Comment (dipesh-rawat) [inline content/hi/docs/reference/glossary/api-eviction.md:7]: Just noticed that the [Netlify build](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/65204e17ca441a00080b6be4) didn't succeed due to an error in Line 7, as indicated in the error message below:

> 7:13:19 PM: Failed during stage "building site": Build script returned non-zero exit code: 2
> 7:13:19 PM: Error: Error building site: "/opt/build/repo/content/hi/docs/reference/glossary/api-eviction.md:7:1": failed to unmarshal YAML: yaml: line 7: could not find expected ":"

Additionally, when viewing the changed [Markdown file](https://github.com/shelar1423/website/blob/main/content/hi/docs/reference/glossary/api-eviction.md) in the GitHub user interface, there seems to be a parsing error, as indicated by the message:

> Error in user YAML: (<unknown>): could not find expected ':' while scanning a simple key at line 6 column 1

It seems that the missing spaces might be causing the errors. Try below suggestion:

```suggestion
short_description: >
  API-eviction निष्कासन वह प्रक्रिया है जिसके द्वारा आप एक बनाने के लिए निष्कासन API का उपयोग करते हैं
```

Comment (shelar1423) [reply content/hi/docs/reference/glossary/api-eviction.md]: sure ,implemented it

**Comment by shelar1423** [comment]
@dipesh-rawat done.

File: content/hi/docs/reference/glossary/api-eviction.md
Comment (dipesh-rawat) [inline content/hi/docs/reference/glossary/api-eviction.md:2]: `title: API-` may require correction as it doesn't align with the corresponding English [file](https://github.com/shelar1423/website/blob/main/content/en/docs/reference/glossary/api-eviction.md?plain=1).

**Comment by shelar1423** [comment]
hey @dipesh-rawat  didnt understand the english file part.  Is this commit correct or what more changes are required if you could brief me a bit more

**Comment by shelar1423** [comment]
@dipesh-rawat changed the title to API -ने निष्कासन शुरू किया

**Comment by aj11anuj** [comment]
If you will ping him in every 2mins then he will kill you, Just kidding 😂

Don't ping him or anybody else.. your PR will get reviewed on its own and you will get timely feedback.

**Comment by shelar1423** [comment]
> If you will ping him in every 2mins then he will kill you, Just kidding 😂
> 
> Don't ping him or anybody else.. your PR will get reviewed on its own and you will get timely feedback.

my bad , sorry , will take care of it from next time

File: content/hi/docs/reference/glossary/api-eviction.md
Comment (aj11anuj) [inline content/hi/docs/reference/glossary/api-eviction.md:2]:
```suggestion
title: एपीआई द्वारा आरंभित निष्कासन (API-Initiated Eviction)
```

File: content/hi/docs/reference/glossary/api-eviction.md
Comment (aj11anuj) [inline content/hi/docs/reference/glossary/api-eviction.md:5]:
```suggestion
full_link: /hi/docs/concepts/scheduling-eviction/api-eviction/
```

File: content/hi/docs/reference/glossary/api-eviction.md
Comment (aj11anuj) [inline content/hi/docs/reference/glossary/api-eviction.md:26]:
```suggestion
* अधिक जानकारी के लिए [एपीआई द्वारा आरंभित निष्कासन](/hi/docs/concepts/scheduling-eviction/api-eviction/) देखें।
```

File: content/hi/docs/reference/glossary/api-eviction.md
Comment (aj11anuj) [inline content/hi/docs/reference/glossary/api-eviction.md:24]:
```suggestion
'एपीआई द्वारा आरंभित निष्कासन', [नोड के दबाव द्वारा निष्कासन](/hi/docs/concepts/scheduling-eviction/node-pressure-eviction/) के समान नहीं है।
```

File: content/hi/docs/reference/glossary/api-eviction.md
Comment (aj11anuj) [inline content/hi/docs/reference/glossary/api-eviction.md:7]:
```suggestion
  'API-Initiated Eviction' निष्कासन की वह प्रक्रिया है, जिसके द्वारा आप एक Eviction ऑब्जेक्ट बनाने के लिए Eviction API का उपयोग करते हैं, जो graceful pod termination को ट्रिगर करती है।
```

File: content/hi/docs/reference/glossary/api-eviction.md
Comment (aj11anuj) [inline content/hi/docs/reference/glossary/api-eviction.md:12]:
```suggestion
'API-Initiated Eviction' निष्कासन की वह प्रक्रिया है, जिसके द्वारा आप एक `Eviction` ऑब्जेक्ट बनाने के लिए [Eviction API](/hi/docs/reference/generated/kubernetes-api/{{<param "version">}}/#create-eviction-pod-v1-core) (निष्कासन एपीआई) का उपयोग करते हैं, जो graceful pod termination को ट्रिगर करती है।
```

File: content/hi/docs/reference/glossary/api-eviction.md
Comment (aj11anuj) [inline content/hi/docs/reference/glossary/api-eviction.md:13]:
```suggestion
```

File: content/hi/docs/reference/glossary/api-eviction.md
Comment (aj11anuj) [inline content/hi/docs/reference/glossary/api-eviction.md:17]:
```suggestion
आप सीधे Eviction API को कॉल करके बेदखली का अनुरोध कर सकते हैं 
Kube-apiserver के क्लाइंट का उपयोग करना, जैसे `kubectl drain` कमांड।
```

File: content/hi/docs/reference/glossary/api-eviction.md
Comment (aj11anuj) [inline content/hi/docs/reference/glossary/api-eviction.md:22]:
```suggestion
API-initiated evictions आपके कॉन्फ़िगर किए गए [`PodDisruptionBudgets`](/hi/docs/tasks/run-application/configure-pdb/) का सम्मान करते हैं
और [`terminationGracePeriodSeconds`](/hi/docs/concepts/workloads/pods/pod-lifecycle#pod-termination)।
```

**Review by aj11anuj — CHANGES_REQUESTED** [review]
Some suggestions from my side

File: content/hi/docs/reference/glossary/api-eviction.md
Comment (aj11anuj) [inline content/hi/docs/reference/glossary/api-eviction.md:15]:
```suggestion
आप सीधे Eviction API को कॉल करके निष्कासन का अनुरोध कर सकते हैं, 
```

File: content/hi/docs/reference/glossary/api-eviction.md
Comment (aj11anuj) [inline content/hi/docs/reference/glossary/api-eviction.md:16]:
```suggestion 
Kube-apiserver के क्लाइंट का उपयोग करके, जैसे `kubectl drain` कमांड।
```

File: content/hi/docs/reference/glossary/api-eviction.md
Comment (aj11anuj) [inline content/hi/docs/reference/glossary/api-eviction.md:20]:
```suggestion
API-initiated evictions आपके कॉन्फ़िगर किए गए [`PodDisruptionBudgets`](/hi/docs/tasks/run-application/configure-pdb/) और 
[`terminationGracePeriodSeconds`](/hi/docs/concepts/workloads/pods/pod-lifecycle#pod-termination) का सम्मान करता है।
```

File: content/hi/docs/reference/glossary/api-eviction.md
Comment (aj11anuj) [inline content/hi/docs/reference/glossary/api-eviction.md:21]:
```suggestion
```

**Review by aj11anuj — CHANGES_REQUESTED** [review]
Some more suggestions from me

**Comment by shelar1423** [comment]
@aj11anuj all good now , ? and also @dipesh-rawat when will this be merged like i want to know what more approval is needed as im new to open source

**Review by aj11anuj — APPROVED** [review]
LGTM, 👍

**Comment by aj11anuj** [comment]
Most of the Reviewers and Approvers have a full-time job at some company and they voluntarily maintain the project when they get time for it. So, they will check your PR on their own. You will get review on your PR quite fast and if your PR is related to localization then approval/merge may take several weeks to even 2-3 months. Don't worry about merge, just address the review feedbacks and rest everything will be fine. Meanwhile, you can make more contributions to the project.

File: content/hi/docs/reference/glossary/api-eviction.md
Comment (satyampsoni) [inline content/hi/docs/reference/glossary/api-eviction.md:20]: Replace `कॉन्फ़िगर` from `नियत `

**Review by satyampsoni — CHANGES_REQUESTED** [review]

**Comment by shelar1423** [comment]
@satyampsoni done

**Comment by satyampsoni** [comment]
Looks good to me!

**Comment by shelar1423** [comment]
Glad for being able to contribute to Kubernetes 😄 ,hopefully this passed the review by @divya-mohan0209 and @Babapool

**Comment by shelar1423** [comment]
Hey @dipesh-rawat  is this final ? and can when this eb merged ?

**Comment by shelar1423** [comment]
Any updates?

**Comment by niranjandarshann** [comment]
@shelar1423  Could you please squash your commits: https://kubernetes.io/docs/contribute/new-content/open-a-pr/#squashing-commits

File: content/hi/docs/reference/glossary/api-eviction.md
Comment (niranjandarshann) [inline content/hi/docs/reference/glossary/api-eviction.md:23]: According to me in the above Sentences For 'API-initiated evictions' We are using the english  So here also 
'API-initiated evictions', [नोड के दबाव द्वारा निष्कासन](/hi/docs/concepts/scheduling-eviction/node-pressure-eviction/) के समान नहीं है। using same will be good for reader to consider. WDYT?

File: content/hi/docs/reference/glossary/api-eviction.md
Comment (niranjandarshann) [inline content/hi/docs/reference/glossary/api-eviction.md:25]: अधिक जानकारी के लिए [API-initiated evictions](/hi/docs/concepts/scheduling-eviction/api-eviction/) देखें।
Here also using `API-initiated evictions` will be good  instead of `एपीआई द्वारा आरंभित निष्कासन`

**Pull request closed without merging**
